engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/jni/JniUtils.java

```diff
@@ -1544,6 +1544,10 @@ public static PtSymbolBlock loadModule(
             String[] extraFileKeys,
             String[] extraFileValues) {
         Device device = manager.getDevice();
+        // MPS doesn't support mapLocation
```

Comment:
@frankfliu  - Could you please add the changes that disable "mapLocation" for "mps" to the master? This will improve the current "mps" support.

Reply:
I created a PR: https://github.com/deepjavalibrary/djl/pull/2061